components/Registration/PlayerForm.tsx

```diff
@@ -0,0 +1,223 @@
+        method: "POST",
+        body: JSON.stringify(playerData)
+      })
+      const createdPlayer = await response.json()
+      await uploadImage(createdPlayer.id)
+      router.reload()
+    } catch (error) {
+      console.log(error)
+    }
+
+  }
+
+  return (
+    <Container maxWidth="md">
+      {new Date().getTime() <
+      new Date(tournament.registrationEndTime).getTime() ? (
+        <>
+          <Box sx={{display: "flex"}}>
+            <Image src={logo} width={60} height={60} alt="logo" />
+            <h1 style={{marginLeft: "10px"}}>Ilmoittautuminen</h1>
+          </Box>
+          <Box sx={{my: 4}}>
+            <p>
+              Tässä lomakkeessa kysytään turnauksen kannalta olennaisia tietoja, jotka näkyvät jahtaajillesi. Lomakkeen täyttämisen jälkeen tuomaristo vahvistaa vielä ilmoittautumisesi.
+            </p>
+            <p>
+              Ainoastaan peitenimi on tässä vaiheessa pakollinen - muut kentät voi täyttää myöhemminkin ja niitä voi muokata turnauksen aikana.
+            </p>
+            <p>
+              Kalenterin tiedot tulee pitää ajan tasalla sekä riittävän selkeinä
+              ja yksityiskohtaisina. Jokaista sekuntia siihen ei tarvitse
+              kirjoittaa, mutta pelistä tulee hauskempaa itsellesi sekä
+              jahtaajillesi jos tarjoat heille riittävästi tilaisuuksia salamurhaamiseen. 
+            </p>
+          </Box>
+          <form>
+            <label>Valitse kuva itsestäsi</label>
+            <input
+              type="file"
+              name="image"
+              accept="image/*"
+              onChange={handleFileInputChange}
+              value={fileInputState}
+            />
+          </form>
+          {selectedFileName ? (
+            <p><i>Valittu tiedosto: {selectedFileName}</i></p>
+          ) : null}
+          <Formik
+            enableReinitialize={true}
+            initialValues={{
+              calendar: [...new Array(dates.length).fill("")],
+              alias: "",
+              title: "noTitle",
+              address: "",
+              learningInstitution: "",
+              eyeColor: "",
+              hair: "",
+              height: 0,
+              glasses: "",
+              other: "",
+            }}
+            validationSchema={Yup.object({
+              alias: Yup.string().required("Pakollinen"),
+            })}
+            onSubmit={(values) => {
+              handleSubmit(values)
+          }}
+          >
+            <Form>
+              <TextInput label="Peitenimi" name="alias" type="text" />
+              <Box sx={{marginBottom: "8px"}}>
+                <div style={{width: "100%"}}>
+                  <label>Ammattilaistitteli</label>
+                </div>
+                <Field name="title" as="select">
+                  <option value="noTitle">Ei titteliä</option>
+                  <option value="KK">KK</option>
+                  <option value="MM">MM</option>
+                  <option value="TT">TT</option>
```

Comment:
Pitäisköhän ennen pitkää pistää pystyyn joku `constants.ts` johon vois hakata näitä titteleitä ja sen sellaisia, jotta niihin voisi viitata vakioina literaaliarvojen sijaan.